projects/packages/forms/src/dashboard/components/actions-dropdown-menu/index.tsx

```diff
@@ -44,12 +47,17 @@ const ActionsDropdownMenu = ( { exportData }: ActionsDropdownMenuProps ) => {
 	}, [ navigate ] );
 
 	const controls = [
+		{
+			icon: plus,
+			onClick: onCreateFormClick,
+			title: __( 'Create form', 'jetpack-forms' ),
+		},
 		{
 			icon: plugins,
 			onClick: onIntegrationsClick,
 			title: __( 'Integrations', 'jetpack-forms' ),
 		},
-		...( exportData.show
+		...( exportData.show && hasItems
```

Comment:
It was suggested we also go ahead and hide the Export item on the mobile menu if there's nothing to export. As part of this, I moved the Create Form button up so Export is the last item. This reduces jumpiness. With the change, the Export button only appears when there are form entries on the current screen. But this also means it's hidden while form entries are loading. 

If export button is higher up, the whole menu jumps as items load or are removed from, say, inbox or spam or trash. 

**Video**
https://github.com/user-attachments/assets/0a067ffe-fde8-40ba-bbd6-7ddf8e06e7cd